components/Markdown/Markdown.css

```diff
@@ -1,55 +1,259 @@
-.msrd .msrd-Markdown h1 {
-  font: var(--font-12);
-  font-weight: 800;
-  letter-spacing: var(--font-12-letter-spacing);
-  margin: 0;
-  margin-bottom: var(--space-vertical-04);
+.msrd .msrd-Markdown {
```

Comment:
How do you feel about knocking the color back slightly to one of our darkest greys?

Comparing to medium / dev.to and they both use slightly offset blacks.

Reply:
Agree, I'm going to take a look at this. IIRC black is used as part of the contrast system and things may start to break (A11Y contrast-wise) if we move it. But agree it's not great for readibility or aesthetics, so will explore.

Reply:
Slightly contstrained by the system, but was able to tweak the two darkest greys (used for backgrounds in dark-mode) to be a bit lighter. It does look better.

From

```css
  --color-grey-0: #111111;
  --color-grey-1: #282828;
```

To

```css
  --color-grey-0: #181818;
  --color-grey-1: #292929;
```

Reply:
I'm not seeing this in the computed styles 🤔 

<img width="546" alt="image" src="https://github.com/measuredco/measuredco-site/assets/985961/ea10a288-cc30-4f53-9b24-df329f5db496">

Reply:
Text color in light-mode is still black, it's the dark-mode backgrounds that have changed

Reply:
Right! I was originally talking about knocking back light-mode text color (not background). This was only a suggestion, so will close this comment unless you'd like to explore otherwise.

Reply:
Ah, black text is required for contrast system, it means we don't have to underline our light-mode links. Still think the dark-mode background change is improvement though! 

<img width="478" alt="Screenshot 2023-10-18 at 11 12 53" src="https://github.com/measuredco/measuredco-site/assets/98794/329b15a6-3bc0-4b0e-ae0d-17f24713a1a6">